google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineSupportProvider.java

```diff
@@ -300,6 +302,10 @@ public void addSupport(@NotNull Module module,
       AppEngineSupportProvider.this
           .addSupport(module, rootModel, myFrameworkSupportModel,
               PersistenceApiComboboxUtil.getSelectedApi(myPersistenceApiComboBox));
+
+      UsageTrackerProvider.getInstance()
+          .trackEvent(GctTracking.APP_ENGINE_SUPPORT_ADDED)
```

Comment:
I believe this will track both new AE projects and AE project imports. What about adding a metric to track usages of the AE standard project wizard? I know there's some overlap there, but it could be nice to have that metric on its own.